src/components/Facts/FactsList.js

```diff
@@ -1,284 +1,151 @@
 import React from 'react';
 import createReactClass from 'create-react-class';
-import Fluxxor from 'fluxxor';
-import ListView from './ListView';
-import { Link } from 'react-router';
-import { SERVICES as DashboardServices } from '../../services/Dashboard';
-import { getHumanDate, containsEqualValues, UCWords, momentLastMonths } from '../../utils/Utils.js';
-import { fragmentToArray, changeFactsUrl } from '../../utils/Fact.js';
-import SuperSelectField from 'material-ui-superselectfield/lib/superSelectField'
 import Chip from 'material-ui/Chip';
+import { PIPELINE_ALL } from '../../actions/constants';
+import { getHumanDate, UCWords } from '../../utils/Utils.js';
+import { methods } from '../../actions/Facts';
+import DialogBox from '../dialogs/DialogBox';
+import ListView from './ListView';
 
 // Material UI style overrides
 const styles = {
   radioButton: {
-    width: "auto",
-    float: "left",
-    marginRight: "20px",
+    width: 'auto',
+    float: 'left',
+    marginRight: '20px',
   },
   checkbox: {
-    marginTop: "6px",
-    marginRight: "20px",
+    marginTop: '6px',
+    marginRight: '20px',
   },
   iconStyle: {
-    marginRight: "4px",
+    marginRight: '4px',
   },
   button: {
-    marginLeft: "5px",
-    height: "20px",
+    marginLeft: '5px',
+    height: '20px',
     lineHeight: '1',
-    textAlign: "center",
+    textAlign: 'center',
   },
   chip: {
-    display: "inline-block",
-    margin: "2px",
+    display: 'inline-block',
+    margin: '2px',
   }
 };
 
-const FluxMixin = Fluxxor.FluxMixin(React);
-const StoreWatchMixin = Fluxxor.StoreWatchMixin("FactsStore");
-
 export const FactsList = createReactClass({
```

Comment:
Rather than using `createReactClass`, could you use `class FactsList extends React.Component` instead?

Reply:
Yes definitely that would be a good change. That's a bigger blast-radius than what I'd like to do right now. Will take a note to do this as a follow-up after we progress this feature.